flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -11,6 +11,7 @@ import AppKit
 public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
     private var sentryOptions: Options?
+    let sentryIOSPrivate = PrivateSentrySDKOnly()
```

Comment:
We can either replace all the calls with `sentryIOSPrivate` now or just use `PrivateSentrySDKOnly` since it's a static class.